packages/vscode-extension/src/webview/utilities/consts.ts

```diff
@@ -24,7 +29,7 @@ export type DeviceProperties = {
 // iOS devices names should match supportedDeviceTypes inside the runtime
 export const iOSSupportedDevices: DeviceProperties[] = [
   {
-    name: "iPhone 15 Pro",
+    displayName: "iPhone 15 Pro",
```

Comment:
we should store the device names in the deviceName variable with the value "iPhone 15 Pro" in this case. The displayName should be determined later based on the deviceName or the user's own device name

Reply:
we decided to store the value `iPhone 15 Pro` in `modelName` 
63f3196c82019876b7f05fbf99e3b924d98643cb